Pipfile

```diff
@@ -3,18 +3,14 @@ name = "pypi"
 url = "https://pypi.org/simple"
 verify_ssl = true
 
-[dev-packages]
-
 [packages]
 docker-compose = ">=1.23.0"
 docker = "*"
 invoke = ">=1.4"
 # https://github.com/docker/compose/issues/6839
 colorama = {version = ">=0.4, <1"}
+pypiwin32 = {version = "*", sys_platform = "== 'win32'"}
+pywin32 = {version = "*", sys_platform = "== 'win32'"}
 
 [requires]
-python_version = "3.7"
-
-[scripts]
```

Comment:
Why removing this part ?
It could be usefull with `pipenv run fab ..`. for legacy projects.

Reply:
This does not work on my machine & Damien machine.
How do you make it work?

Reply:
It's only working with pipenv run, did you try pipenv shell ?

Reply:
Done